docs/Developers-Guide.md

```diff
@@ -4,12 +4,12 @@
 
 _A Quick guide to setting up the OpenTripPlanner project._
 
-You need Git, Maven and Java(JDK) and an IDE installed on your computer. You IDE might have JDK and
+You need Git, Maven and Java 17 and an IDE installed on your computer. Your IDE might have JDK and
```

Comment:
I suggest keeping "Java(JDK)". If we upgrade Java later we might forget to update this. We hava some support cases where people uses the wrong version of Java, but my impression is that documenting it here want help. It is documented in the maven pom file, that should be enough.